### PR TITLE
Update appdev/nativeapp/index.rst

### PR DESCRIPTION
**Pull request by rocketkittens**
Spelling correction and wording change.

### CODE REVIEW

File: appdev/nativeapp/index.rst [hunk 1]
```diff
@@ -4,11 +4,11 @@ Native applications
 ===================
 
 The recommended way to build a native application on Ubuntu Touch is to use the `Qt <https://doc.qt.io/>`__ framework ( current version shipped on Ubuntu Touch 16.04 is Qt 5.9.5 ).
-You are not stick with the traditionnal C++<->QML Qt model, you can use also Python, Go, Rust or even Javascript for your backend.
+You are not stuck with the traditionnal C++<->QML Qt model, you can use also Python, Go, Rust or even Javascript for your backend.
 
-Ubuntu Touch provide a subset of Qt QML components to ease you design your app : `QML API <https://api-docs.ubports.com/sdk/apps/qml/index.html>`__ 
+Ubuntu Touch provide a subset of Qt QML components to ease the developement your app : `QML API <https://api-docs.ubports.com/sdk/apps/qml/index.html>`__
```
Comment (UniversalSuperBox) [inline appdev/nativeapp/index.rst]: the development *of*

File: appdev/nativeapp/index.rst [hunk 2]
```diff
@@ -4,11 +4,11 @@ Native applications
...
 
-Ubuntu Touch provide a subset of Qt QML components to ease you design your app : `QML API <https://api-docs.ubports.com/sdk/apps/qml/index.html>`__ 
+Ubuntu Touch provide a subset of Qt QML components to ease the developement your app : `QML API <https://api-docs.ubports.com/sdk/apps/qml/index.html>`__ 
 and Icons: `Suru Icons <http://docs.ubports.com/projects/icons/en/latest/>`__, but you can also ship your cross platform app using standard Qt QML QtQuick2 components.
 
 You can use `Clickable <http://clickable.bhdouglass.com/en/latest/>`_ to facilitate Packaging, deploying, publishing and tests .
 
-Check out informations about the platform and confinement model in :ref:`Platform <platform>` section 
\ No newline at end of file
+Check out informations about the platform and confinement model in :ref:`Platform <platform>` section
```
Comment (UniversalSuperBox) [inline appdev/nativeapp/index.rst]: Do you know what changed in this line? "Informations" should be "information"

Comment (UniversalSuperBox) [reply appdev/nativeapp/index.rst]: I figured out what changed, a final newline was added.

Another change to this file has caused this pull request to conflict, could you fix the conflicts? You can either do it manually if you know git pretty well or create a new pull request by following the same steps you followed to make this one.

**Comment by Flohack74** [comment]
@rocketkittens are you still up to getting this resolved?

**Comment by Flohack74** [comment]
@rocketkittens are you still up to getting this resolved?

**Comment by doniks** [comment]
stalled here. lets get this over with: https://github.com/ubports/docs.ubports.com/pull/284

**Pull request closed without merging**